src/FileProviders/Embedded/src/Microsoft.Extensions.FileProviders.Embedded.csproj

```diff
@@ -12,6 +12,7 @@
     <PackageTags>files;filesystem</PackageTags>
     <NoPackageAnalysis>true</NoPackageAnalysis>
     <Nullable>enable</Nullable>
+    <IsTrimmable>true</IsTrimmable>
```

Comment:
Based on the [docs on configuring trimming](https://github.com/dotnet/aspnetcore/blob/bec278eabea54f63da15e10e654bdfa4168a2479/docs/Trimming.md#trim-an-aspnet-core-assembly) I believe that we should also see an update to [eng/TrimmableProjects.props](https://github.com/dotnet/aspnetcore/blob/bec278eabea54f63da15e10e654bdfa4168a2479/eng/TrimmableProjects.props) following this change.